src/app/components/InlineTextArea/styles.module.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../../scss/index.scss';
+
+.spanFontSize {
+  font-size: $small;
+}
+.spanFontSizeMedium {
+  font-size: $large;
+}
+.spanContainer {
+  width: 30rem;
```

Comment:
we usually don't use rem... actually, we use it only 2 times in the entire virtual office. (Is not a requested change. There is no problem to use it here but to be in line with the team, we usually avoid it)